uditagrawal08/task8.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Task 8</title>
+    <style>
+    .container {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 100vh;
+        
+    }
+    
+    .container img{
+          transition: width 1s;
+        width: 50%;
+    }
+    
+    .container img:hover{
+        border: 10px solid rosybrown;
+        border-radius: 5%;
+        width: 400px;
+        cursor: pointer
+
+      }
+    .modal{
+
+      display:none;
+      position: fixed;
+    
+      padding: 100px;
+      top: 0;
+      width: 100%;
+      height: 100%;
+      overflow: auto;
+      
+    }
+    .modal-content{
+      text-align: left;
+      display: block;
+      width: 75%;
+      height: 30%;
+      border-radius: 20px;
+      background-color:aliceblue;
+      padding: 20px;
+      box-shadow:0px 8px 8px 0px violet
+    }
+      #close{
+       background-color: darkgray; 
+       float: right;
+       margin-left: 30px;
+       font-size: large;
+       border-radius: 10px;
+
+      }
+      #download{
+        background-color:darkgray;
+        float: right;
+        height: 50%;
+        font-size: large;
+        border-radius: 10px;
+      }
+      .modal-content button:hover{
+        cursor:pointer;
+      }
+    </style>
+</head>
+<body>
+
+  <div class="container">
+    <img id="img" src="/web-development/assets/images/task8.jpg" alt="this is image">
+  </div>
+  <div id="my-modal" class="modal">
+  <div class="modal-content">
+    <h1>RED DAHLIA FLOWER</h1>
+    <br>
+    <p>This is a flower that exites your eyes with beauty and pattern of its red petals. </p>
+    <div class="close">
+    <button onclick="closeFunction()" id="close">Close</button>
+    </div>
+    <div class="download">
+      <a href="/web-development/assets/images/task8.jpg" download>
```

Comment:
This is an absolute path which will not work on other systems.
Try adding relative path `../assets/images/task8.jpg`.